src/lib/vueify.js

```diff
@@ -91,7 +91,10 @@ function generateModel(el, data) {
  * @returns {boolean|string|null}
  */
 function generateModelForInput(el) {
-  if (el.getAttribute('type') === 'checkbox') {
+  if (
+    el.getAttribute('type') === 'checkbox' ||
+    el.getAttribute('type') === 'radio'
```

Comment:
Is this a better way to do it?  unless `.includes()` is available.
```suggestion
    ['checkbox', 'radio'].indexOf(el.getAttribute('type')) >= 0
```

Reply:
good call.